styles/index.css

```
@@ -209,13 +233,15 @@ html.dark {
   --marker-color: #bc3553;
 }
 .alchemmist-section-marker.is-violet {
-  --marker-color: #7651dc;
+  --marker-color: #4724a8;
 }
 
 .alchemmist-card {
   --card-accent: var(--alchemmist-accent);
   background: var(--alchemmist-surface-elevated);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)